### PR TITLE
Add insecure option to the automate report json

### PR DESCRIPTION
**Pull request by jquick**
Currently the automate report json uses `verify_ssl` while the compliance json uses `insecure` this is confusing. Adding in the `insecure` as a option for automate and will use that in the documentation.

Signed-off-by: Jared Quick <jquick@chef.io>

### CODE REVIEW

**Review by clintoncwolfe — COMMENTED** [review]
I didn't see a documentation change.  Is there intended to be one?

**Comment by jquick** [comment]
Yes, we finalized the style so I will add some docs to this PR.

File: docs/reporters.md [hunk 1]
```diff
@@ -115,3 +115,47 @@ This reporter includes all information from the rspec runner. Unlike the json re
...
+        "url" : "https://YOUR_A2_URL/data-collector/v0/",
+        "token" : "YOUR_A2_ADMIN_TOKEN",
+        "insecure" : true,
+        "node_name" : "inspec_test_node",
+        "environment" : "prod"
+    }
+}
+```
+
+### Manditory fields:
```
Comment (clintoncwolfe) [inline docs/reporters.md]: mandatory

**Review by clintoncwolfe — APPROVED** [review]
Just one spello.  Thanks @jquick!

**Pull request merged**